app/src/main/java/com/android/sample/model/profile/ProfileViewModel.kt

```diff
@@ -124,6 +118,11 @@ constructor(private val repository: ProfilesRepository, private val localDatabas
     repository.updateProfile(user = user, onSuccess = { fetchUserData(user.id) }, onFailure = {})
   }
 
+  // I believe this function belongs to signInRepository (or maybe viewModel) instead of
+  // profileViewModel
+  // because it uses auth.currentUser
+  // I would move this function to SignInRepository if I wanted to test it in the e2e tests
+  // Written by: @mohamedtahaguelzim
```

Comment:
Please ignore this comment for the moment. Will investigate further on it later.